### DISCUSSION TITLE
is there Intel Daos support future plan ?

### DISCUSSION THREAD

**Discussion by yyyshi** [rook/rook#12451] [Q&A]
is there Intel Daos support future plan ?

**Comment by travisn** [comment]
Please explain more. What integration do you see is needed? Rook is expected to run in any K8s cluster as long as [raw block storage devices](https://rook.io/docs/rook/latest/Getting-Started/Prerequisites/prerequisites/) are available.

**Comment by cheese** [comment]
It's good to see that. Daos needs to be more integrated with k8s.
